### DISCUSSION TITLE
Be able to download the full description of a Youtube video

### DISCUSSION THREAD

**Discussion by formlessdao** [DominikPieper/obsidian-ReadItLater#73] [General]
Hi, can you add please the ability to use a %videodescription% so the full video description is also saved into Obsidian?
Thank you in advance!

**Comment by thorms11** [comment]
I’d love to have this as well!

**Comment by adamluckdev** [comment]
Right now plugin parse HTML markup of YouTube that's because it has limited set of variables. More details about video can be fetched via API, but it requires using key which is linked to concrete Google account. 

So, the solution will be new setting for this key. If it's set, details about video will be fetched from API. Hopefully, it'll be included in next release.

**Comment by adamluckdev** [comment]
Released in [0.2.0](https://github.com/DominikPieper/obsidian-ReadItLater/releases/tag/0.2.0)
